src/routes/tags.js

```diff
@@ -0,0 +1,13 @@
+const Router = require("koa-router");
+const Tags = require("../db/tags");
+
+module.exports = class Tags extends Router {
+    constructor(app){
+        super();
+        this.get("/tags", this.getTags);
+    }
+    async getTags(ctx){
+        const tags = await Tags.find({});
```

Comment:
You can reference the *src/passport.js* file for an example. `Tags` here should be a `db.collection`.